geopandas/tests/test_geodataframe.py

```diff
@@ -361,6 +361,14 @@ def test_mixed_types_to_file(self):
         with pytest.raises(ValueError):
             s.to_file(tempfilename)
 
+    def test_empty_to_file(self):
+        input_empty_df = GeoDataFrame()
+        tempfilename = os.path.join(self.tempdir, 'test.shp')
+        with pytest.raises(
+            ValueError, message="Cannot write empty DataFrame to file."):
```

Comment:
to test the correct message, I think you need to use the `match=` keyword (the `message` keyword is to give a custom message if the *assert* failes, somewhat confusing I have to say ..)

Reply:
Good catch. Pytest is really powerful but a little confusing at times :)